examples/getting-started-app/getting-started-app-scm-1.rockspec

```diff
@@ -7,7 +7,7 @@ source  = {
 dependencies = {
     'tarantool',
     'lua >= 5.1',
-    'luatest == 0.4.0-1',
+    'luatest == 0.5.0-1',
     'ldecnumber == 1.1.3-1',
     'cartridge == 1.2.0-1',
```

Comment:
Let's bump it to 2.0.1

Reply:
It should be updated along with getting started readme